tests/x11/reboot_gnome.pm

```diff
@@ -40,7 +40,8 @@ sub run() {
 
         send_key "ret";
 
-        if (check_screen 'reboot-auth', 5) {
+        wait_still_screen;    # wait 7 seconds to make authentication window disappear after successful authentication
```

Comment:
won't the `wait_still_screen` fail when the reboot actually happens and the screen keeps on changing?